docs/self-hosted-appcircle/install-server/helm-chart/upgrades.md

```diff
@@ -7,7 +7,21 @@ sidebar_position: 30
 
 import NeedHelp from '@site/docs/\_need-help.mdx';
 
```

Comment:
- [ ] Add version history table to the Uprade page as we did before for the standalone arch.